components/Workspaces/RepositoryStatCard.tsx

```diff
@@ -91,11 +103,13 @@ export const RepositoryStatCard = ({ stats, type, isLoading }: RepositoryStatCar
                 </th>
                 {stat === "health" ? (
                   <td className="text-black semi-bold text-2xl">
-                    {value}
+                    {Math.round(value)}
                     <span className="text-xs">/10</span>
                   </td>
                 ) : (
-                  <td className="semi-bold text-2xl">{stat === "velocity" ? `${Math.round(value)}d` : value}</td>
+                  <td className="semi-bold text-2xl" title={`${value}`}>
+                    {stat === "velocity" ? `${Math.round(value)}d` : humanizeNumber(value, "abbreviation")}
```

Comment:
We're humanizing the numbers here because in some cases, like Kubernetes, the numbers are huge.

![image](https://github.com/open-sauced/app/assets/833231/b4e071b9-5bfb-43de-bfea-43883a34ad81)